docs/coding-guidelines/breaking-change-rules.md

```diff
@@ -16,7 +16,7 @@ Breaking Change Rules
 
 ## Behavioral Changes
 
-### Property, Field, Parameter and Return Values
+### Method, Property, Field, Parameter and Return Values
```

Comment:
This change probably needs a corresponding change on line 5 (both the URL and the TOC entry)